ecommerce/utils.py

```diff
@@ -108,3 +109,26 @@ def validate_amount(discount_type, amount):
 def positive_or_zero(number):
     """Return 0 if a number is negative otherwise return number"""
     return 0 if number < 0 else number
+
+
+class CouponUtils:
+    @staticmethod
+    def validate_unique_coupon_code(value):
+        """
+        Validates the uniqueness of coupon codes in Coupon and B2BCoupon models.
```

Comment:
This is a linting error in Ruff. We are working on integrating it.
```suggestion
        Validate the uniqueness of coupon codes in Coupon and B2BCoupon models.
```